nilearn/datasets/tests/test_func.py

```diff
@@ -32,47 +29,55 @@ def _load_localizer_index():
         localizer_template = json.load(of)
     localizer_index = {}
     for idx in range(1, 95):
-        sid = 'S{:02}'.format(idx)
-        localizer_index.update(dict(
-            (key.format(sid), uuid.uuid4().hex)
-            for key in localizer_template))
-    localizer_index['/localizer/phenotype/behavioural.tsv'] = uuid.uuid4().hex
-    localizer_index['/localizer/participants.tsv'] = uuid.uuid4().hex
+        sid = f"S{idx:02}"
+        localizer_index.update(
+            {key.format(sid): uuid.uuid4().hex for key in localizer_template}
+        )
+    localizer_index["/localizer/phenotype/behavioural.tsv"] = uuid.uuid4().hex
+    localizer_index["/localizer/participants.tsv"] = uuid.uuid4().hex
     tsv_files = {}
-    tsv_files['/localizer/phenotype/behavioural.tsv'] = pd.read_csv(
-        str(data_dir / 'localizer_behavioural.tsv'), sep='\t')
-    tsv_files['/localizer/participants.tsv'] = pd.read_csv(
-        str(data_dir / 'localizer_participants.tsv'), sep='\t')
+    tsv_files["/localizer/phenotype/behavioural.tsv"] = pd.read_csv(
+        str(data_dir / "localizer_behavioural.tsv"), sep="\t"
```

Comment:
We don't need to cast to string here:

```suggestion
        data_dir / "localizer_behavioural.tsv", sep="\t"
```